docs/website/src/main/resources/certifications/jakarta-platform/11/TCK-Results-8.0.0-M12.md

```diff
@@ -402,6 +402,7 @@ Number of Tests with Errors = 0
 Job name: jakarta-signature-tck-glassfish
 
 ```
```

Comment:
why Tests run is here twice?

Reply:
It's a very complex technicality of the signature test having multiple executions for various protocols. @scottmarlow would be the one to truly ask this too. I just report the results here.

Reply:
@arjantijms @OndroMih good question!  Each spec api is checked in both `static mode` and `reflective mode`.  The `reflective mode` testing validates that the expected SPEC APIs are available to EE application deployments in the EE containers.

The `static mode` is a non-EE test that checks a statically specified classpath setting.  If you configure that incorrectly, you will see failures.  If one of your SPEC APIs is a super/sub set you will see both `static mode` +`reflective mode` failures.

```
jakarta.persistence.criteria(static mode)
jakarta.persistence.criteria(reflection mode)
```